terragrunt/aws/api/lambda.tf

```diff
@@ -0,0 +1,32 @@
+module "url_shortener_lambda" {
+  source                 = "github.com/cds-snc/terraform-modules?ref=v5.0.0//lambda"
+  name                   = "${var.product_name}-api"
+  billing_tag_value      = var.billing_code
+  ecr_arn                = aws_ecr_repository.api.arn
+  enable_lambda_insights = true
+  image_uri              = "${aws_ecr_repository.api.repository_url}:latest"
+  memory                 = 3008
+  timeout                = 300
+  ephemeral_storage      = 768
```

Comment:
We can get rid of this since the default `ephemeral_storage` size of the lambda will be enough for us.

Reply:
Done!